patch/PCI-ASPM-Fix-link-state-exit-during-switch-upstream.patch

```diff
@@ -0,0 +1,77 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Daniel Stodden <daniel.stodden@gmail.com>
+Date: Sun, 22 Dec 2024 19:39:08 -0800
+Subject: [PATCH] PCI/ASPM: Fix link state exit during switch upstream function
+ removal
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
```

Comment:
Please add the upstream commit hash at the beginning.

[ Upstream commit cbf937dcadfd571a434f8074d057b32cd14fbea5 ]